specification/columns/commitmentdiscountquantity.md

```diff
@@ -10,17 +10,17 @@ The CommitmentDiscountQuantity column adheres to the following requirements:
 
 * CommitmentDiscountQuantity MUST be present in a FOCUS dataset when the provider supports *commitment discounts*.
 * CommitmentDiscountQuantity MUST be of type Decimal and MUST conform to [Numeric Format](#numericformat) requirements.
-* CommitmentDiscountQuantity MAY be negative if [*ChargeClass*](#chargeclass) is "Correction".
+* CommitmentDiscountQuantity MAY be null or any valid decimal value if [*CommitmentDiscountId*](#commitmentdiscountid) is not null and [*ChargeClass*](#chargeclass) is "Correction".
```

Comment:
```suggestion
* CommitmentDiscountQuantity MAY be null or any valid decimal value if *ChargeCategory* is "Purchase" or "Usage", [*CommitmentDiscountId*](#commitmentdiscountid) is not null, and [*ChargeClass*](#chargeclass) is "Correction".
```

Reply:
A non-null commitment discount id  always references either a Purchase or Usage, so I don't think this is needed.

Reply:
Considering that some dataset producers might interpret the requirements individually and assume that, in case of corrections, it’s acceptable to provide a value regardless of Charge Category, as long as the CommitmentDiscountId is included, I prefer to be as precise as possible.

Reply:
> A non-null commitment discount id always references either a Purchase or Usage

I assumed the same, but when I checked, I realized that we didn’t specify that. Therefore, I suggest we include those two conditions as well.

Reply:
This addition does not add anything to make this statement more correct or incorrect.